### DISCUSSION TITLE
Blender agx

### DISCUSSION THREAD

**Discussion by mokranecg** [sobotka/AgX#5] [Q&A] [answered]
How do you set texture color space Diffuse/Height/Roughness/HDRI color space.
Is there any documentation on how to work with agx in blender?

**Comment by EaryChow** [comment, marked as answer]
The textures you originally set to `sRGB` would remain as sRGB, the ones set to `Linear` is now `Linear BT.709`, the original `raw` or `non-color` is now `Generic Data`.
